IBPSA/Airflow/Multizone/BaseClasses/TwoWayFlowElement.mo

```diff
@@ -9,8 +9,16 @@ partial model TwoWayFlowElement "Flow resistance that uses the power law"
     final m2_flow_nominal=m1_flow_nominal);
   extends IBPSA.Airflow.Multizone.BaseClasses.ErrorControl;
 
-  replaceable package Medium = Modelica.Media.Interfaces.PartialMedium
-    annotation (choicesAllMatching=true);
+  replaceable package Medium =
+    Modelica.Media.Interfaces.PartialMedium "Medium in the component"
+      annotation (choices(
+        choice(redeclare package Medium = IBPSA.Media.Air "Moist air"),
+        choice(redeclare package Medium = IBPSA.Media.Water "Water"),
+        choice(redeclare package Medium =
```

Comment:
We could add an air medium that includes CO2?

Reply:
@Mathadon : I suggest to keep it at these 3 mediums for now, and add it if we see a clear need.